Documentation improvements (#145)
Stormbase/django-otp-webauthn: docs/how_to_guides/index.rst

   :gutter: 3
   :margin: 0

   .. grid-item-card:: :ref:`Sharing passkeys between domains <configure_related_origins>`

>        Learn how to configure WebAuthn to work across multiple domains. For example, if your main application runs on ``https://example.com`` and you have a localized version on ``https://example.co.uk``.

Reviewer: ```suggestion
   .. grid-item-card:: :ref:`Share passkeys between domains <configure_related_origins>`

       Learn how to configure WebAuthn to work across multiple domains. For example, if your main application runs on ``https://example.com`` and you have a localized version on ``https://example.co.uk``.
```

>    .. grid-item-card:: :ref:`Identifying Passkeys <howto-identifying-passkeys>`

Reviewer: ```suggestion
   .. grid-item-card:: :ref:`Identify passkeys <identify-passkeys>`
```

Author: ```suggestion
   .. grid-item-card:: :ref:`Identify passkeys <howto-identify-passkeys>`
```

>        Learn how to identify the authenticator that created a Passkey, for example Apple Passwords, Yubikey, Android etc. This is useful context to give your users when presenting them with a list of their registered Passkeys.

Reviewer: ```suggestion
       Learn how to identify the authenticator that created a passkey. For example, it can identify Apple Passwords, YubiKey, and Android. This is useful context to give your users when presenting them with a list of their registered passkeys.
```

       Learn how to create custom Django OTP WebAuthn models if the base models don't suit your needs.

.. toctree::
    :maxdepth: 2
    :hidden:

>     Sharing passkeys between domains <configure_related_origins.rst>

Author: I've renamed this t conform to the goal a developer might have in mind when reading this ('how do I make this work across domains?')

I also moved it up since it is a pretty important feature, at least more important than the other customization sections.

       Learn how to create custom Django OTP WebAuthn models if the base models don't suit your needs.

   .. grid-item-card:: :ref:`Keeping Passkeys up-to-date with changing user details <keeping_passkeys_in_sync>`

       Learn how to keep Passkey user details saved in users' browsers up-to-date when details like email or username change.

.. toctree::
    :maxdepth: 2
    :hidden:

>     Sharing passkeys between domains <configure_related_origins.rst>

Reviewer: ```suggestion
    Share passkeys between domains <configure_related_origins.rst>
```

>     Identifying Passkeys <identifying_passkeys.rst>

Reviewer: ```suggestion
    Identify passkeys <identifying_passkeys.rst>
```

    Customize views <customize_views.rst>
    Customize helper class <customize_helper_class.rst>
    Customize models <customize_models.rst>

>     Keeping Passkeys up-to-date with changing user details <keeping_passkeys_in_sync.rst>

Reviewer: ```suggestion
    Keeping passkeys up-to-date with changing user details <keeping_passkeys_in_sync.rst>
```
